src/types.ts

```diff
@@ -149,5 +150,6 @@ export type NativeMenuComponentProps = {
   style?: StyleProp<ViewStyle>;
   onPressAction?: ({ nativeEvent }: NativeActionEvent) => void;
   actions: ProcessedMenuAction[];
+  actionsHash: string;
```

Comment:
[question] is this change required for the fabric architecture? if not, I'd like to keep it separate from this PR